src/node.cc

```diff
@@ -3111,6 +3111,7 @@ static void PrintHelp() {
          "  --track-heap-objects  track heap object allocations for heap "
          "snapshots\n"
          "  --v8-options          print v8 command line options\n"
+         "  --cipher-list=list    use an alternative default cipher list\n"
```

Comment:
This should be clarified a bit to say that it only affects TLS and does not limit what ciphers can be used with the `crypto` module.